### PR TITLE
Add 3.10 arm64 helix image

### PR DESCRIPTION
**Pull request by jashook**
/cc @safern @lpereira @trylek

### CODE REVIEW

**Comment by jashook** [comment]
@MichaelSimons ptal

/cc @janvorli

File: manifest.json [hunk 1]
```diff
@@ -398,6 +398,20 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/alpine/3.10/helix/arm64v8",
+              "os": "linux",
+              "osVersion": "alpine3.8",
```
Comment (mthalman) [inline manifest.json]:
```suggestion
              "osVersion": "alpine3.10",
```

File: src/alpine/3.10/helix/arm64v8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+# Alpine does not support long options
+RUN /usr/sbin/adduser -D -g '' -G adm -s /bin/bash -u 1001 helixbot && \
+    chmod 755 /root && \
+    echo "helixbot ALL=(ALL)       NOPASSWD: ALL" > /etc/sudoers.d/helixbot
+
+USER helixbot
+
+RUN python -m virtualenv --no-site-packages /home/helixbot/.vsts-env
+
+
```
Comment (mthalman) [inline src/alpine/3.10/helix/arm64v8/Dockerfile]: Remove extra line

**Comment by jashook** [comment]
@mthalman I believe the feedback is addressed

**Pull request merged**